x-pack/plugin/esql/qa/testFixtures/src/main/resources/categorize.csv-spec

```diff
@@ -218,12 +230,11 @@ FROM hosts
 ;
 
 COUNT():long | category:keyword
-           2 | .*?DB.+?servers.*?
```

Comment:
`DB` is parsed as a hex number and dropped. Obviously suboptimal, but unrelated to this PR.